diff -Naur holoviews-1.10.9.orig/holoviews/plotting/bokeh/callbacks.py holoviews-1.10.9/holoviews/plotting/bokeh/callbacks.py
--- holoviews-1.10.9.orig/holoviews/plotting/bokeh/callbacks.py	2018-10-30 10:21:28.000000000 -0300
+++ holoviews-1.10.9/holoviews/plotting/bokeh/callbacks.py	2018-11-06 11:45:49.974025874 -0200
@@ -777,9 +777,9 @@
     Returns the current selection on a ColumnDataSource.
     """
 
-    attributes = {'index': 'cb_obj.selected.indices'}
-    models = ['source']
-    on_changes = ['selected']
+    attributes = {'index': 'cb_obj.indices'}
+    models = ['selected']
+    on_changes = ['indices']
 
     def _process_msg(self, msg):
         if 'index' in msg:
diff -Naur holoviews-1.10.9.orig/holoviews/plotting/bokeh/element.py holoviews-1.10.9/holoviews/plotting/bokeh/element.py
--- holoviews-1.10.9.orig/holoviews/plotting/bokeh/element.py	2018-10-30 10:21:28.000000000 -0300
+++ holoviews-1.10.9/holoviews/plotting/bokeh/element.py	2018-11-06 11:45:25.517403648 -0200
@@ -711,6 +711,7 @@
             source = self._init_datasource(data)
         self.handles['previous_id'] = current_id
         self.handles['source'] = self.handles['cds'] = source
+        self.handles['selected'] = source.selected
 
         properties = self._glyph_properties(plot, style_element, source, ranges, style)
         with abbreviated_exception():
